### ISSUE TITLE
RightToLeft doesn't work

### ISSUE THREAD

**Issue by EQuimper** [react-native-text-input-mask/react-native-text-input-mask#206]
Hello thanks for this library. I did see one of the last release rightToLeft props was add. If I set it to true this doesn't work for me. Look like it's always leftToRight. Not sure if you do have the same issue.

**Comment by martintreurnicht** [comment]
@EQuimper Yes this was exposed recently as it was requested by several others although it hasn't been tested. I haven't looked into how this property is used on the native side exactly, I'll have a look at it later this week to try and figure out why it's not working

**Comment by EQuimper** [comment]
Sounds good @martintreurnicht thank you :)

**Comment by NathHorrigan** [comment]
Hi, @martintreurnicht  - Did you get around to investigating this?

**Comment by kudresov** [comment]
Have the same issue here, after quick at the code aren't we missing `rightToLeft` to be passed in here: https://github.com/react-native-text-input-mask/react-native-text-input-mask/blob/530a1b6a5e09f64d83f7bb2cd6740a7132794d2d/ios/TextInputMask.swift#L48 ?

_Disclaimer: not an iOS dev, so might be talking complete nonsense_

**Comment by whatoken** [comment]
Same issue, any update on this?
